projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/quick-order/b2b-quick-order.e2e-spec.ts

```diff
@@ -0,0 +1,91 @@
+import * as quickOrder from '../../../../helpers/b2b/b2b-quick-order';
+import { viewportContext } from '../../../../helpers/viewport-context';
+import * as sampleData from '../../../../sample-data/b2b-checkout';
+
+context('B2B - Quick Order', () => {
+  viewportContext(['mobile', 'desktop'], () => {
+    before(() => {
```

Comment:
change this to beforeEach, and you can skip adding `quickOrder.clearList` in line 21, 30,40

Reply:
I cannot do it as for other tests in this 'describe' I cannot call clear list it will crash

Reply:
Have you tried it? 

What you have inside this before removes all session, cookies, storage, etc
Basically, it resets the app to be like in a vanilla state.

You can verify it gets trigged if you put a log to see it firing for each of the `it`.

Also, don't forget to try it out as this solution is better than clearing the list for each new `it`. We want the test to not be dependent on each other as what is happening right now.